user_guide_src/source/libraries/caching.rst

```diff
@@ -36,14 +36,14 @@ $handler
 ========
 
 The is the name of the handler that should be used as the primary handler when starting up the engine.
-Available names are: dummy, file, memcached, redis, predis, wincache.
+Available names are: dummy, file, memcached, redis, predis, wincache. This is commonly the **File** handler
+since the file system is always available, but may not fit more complex, multi-server setups.
```

Comment:
Why do we need "This is commonly the **File** handler
since the file system is always available, but may not fit more complex, multi-server setups." here?

Reply:
@kenjis I didn't add. It is already there.

Reply:
No. See https://codeigniter4.github.io/CodeIgniter4/libraries/caching.html#handler
Please remove the change.

Reply:
@kenjis Sorry. Removed.